### PR TITLE
Update the French translation

### PR DESCRIPTION
**Pull request by Fabrice-TIERCELIN**
Hi @IvanSavenko,

This PR updates the French translations. It also transfers the last documentation of the VCMI wiki to the project and creates the first version of the Swedish translation.

### CODE REVIEW

File: docs/modders/Entities_Format/Creature_Help.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# Creature Help
+
+This page helps you to create a creature (i.e. a unit that fights in a battle) for a creature mod or inside a bigger mod like a [faction mod](Faction_Help.md).
+
+## Utilities
+
+You need to download the two utilities `DefPreview` and `H3DefTool` from the internet:
+- `DefPreview` converts a `.def` file to `.bmp` images
+- `H3DefTool` converts `.bmp` images to a `.def` file
```
Comment (IvanSavenko) [inline docs/modders/Entities_Format/Creature_Help.md:9]: Perhaps add download link?

File: docs/modders/Entities_Format/Creature_Help.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+| c | c | c | c | c | c | c | c | c | c |
+| c | c | c | c | c | c | c | b | c | c |
+| c | c | c | c | c | c | c | b | c | c |
+| c | c | c | c | c | c | b | b | b | c |
+| c | c | c | c | c | c | b | b | b | c |
+| c | c | c | c | c | b | b | b | b | b |
+| c | c | c | c | c | c | b | b | b | c |
+| c | c | c | c | c | c | c | c | c | c |
+| c | c | c | c | c | c | c | c | c | c |
+| c | c | c | c | c | c | c | c | c | c |
```
Comment (IvanSavenko) [inline docs/modders/Entities_Format/Creature_Help.md:98]: perhaps remove 'c''s or replace them with single dot to make this table more easy to read?

**Comment by Fabrice-TIERCELIN** [comment]
@IvanSavenko, I suggest to split the documentations for the modders and for the translators. They are not the same persons.

**Comment by IvanSavenko** [comment]
If you wish to improve docs - that would be great, feel free to make them more easy to use. I wrote them from my developer point of view, so yeah, there might be parts that sound obvious to me, but are unclear to newcomers.

As for splitting - I am not against it, but how would that improve docs? I mean, right now we have 1 page about translation. Moving it to another location won't change much. Unless you're planning to extend them.

**Comment by Fabrice-TIERCELIN** [comment]
@IvanSavenko, the Swedish translation is not taken into account. Would it be possible to fix the language code? I think that `sv_SV` should be changed to `sv_SE`, isn't it?

**Comment by IvanSavenko** [comment]
https://github.com/vcmi/vcmi/pull/4607

**Comment by Fabrice-TIERCELIN** [comment]
Hi @XCOM-HUB & @CalZing,

I have created a Swedish translation for the launcher of VCMI. I have filled the 280 entries with _Google translate_. If you want, you can proofread the entries in [this file](https://github.com/vcmi/vcmi/blob/develop/launcher/translation/swedish.ts). You can see what it gets in the launcher by installing the [latest develop version of VCMI](https://builds.vcmi.download/branch/develop/Windows/).

You can create a PR with the new version but you can also send me the fixed version and I can do the PR for you.

I have also created an [empty translation project](https://github.com/vcmi-mods/empty-translation) to find the images without previous captions.

PS: I'm creating the "LADDA" button:
state

**Comment by Laserlicht** [comment]
@Fabrice-TIERCELIN For the [empty-translation](https://github.com/vcmi-mods/empty-translation) and the newgame & loadgame I suggest you to use transparent images. No need for copying background (advantage: it's also work for e.g. SoD background). Also for the separate buttons if you plan to do some of them ([see themes mod](https://github.com/vcmi-mods/h3-themes/tree/vcmi-1.5/h3-themes/mods/AB/content/sprites/images)).

**Comment by Fabrice-TIERCELIN** [comment]
OK I will do translations using transparency. For now, in the `empty-translation`, I will leave the images because a transparent rectangle won't help nobody.

Are there text-empty images somewhere?

**Comment by XCOM-HUB** [comment]
> @IvanSavenko, the Swedish translation is not taken into account. Would it be possible to fix the language code? I think that `sv_SV` should be changed to `sv_SE`, isn't it?

Correct! It should be sv-SE

https://simplelocalize.io/data/locale-code/sv-SE/

**Comment by XCOM-HUB** [comment]
> Hi @XCOM-HUB & @CalZing,
> 
> I have created a Swedish translation for the launcher of VCMI. I have filled the 280 entries with _Google translate_. If you want, you can proofread the entries in [this file](https://github.com/vcmi/vcmi/blob/develop/launcher/translation/swedish.ts). You can see what it gets in the launcher by installing the [latest develop version of VCMI](https://builds.vcmi.download/branch/develop/Windows/).
> 
> You can create a PR with the new version but you can also send me the fixed version and I can do the PR for you.
> 
> I have also created an [empty translation project](https://github.com/vcmi-mods/empty-translation) to find the images without previous captions.
> 
> PS: I'm creating the "LADDA" button: state

@Fabrice-TIERCELIN - Did you translate "LOAD GAME" to "LADDA"? "LOAD GAME" should be translated as "LADDA SPEL". Only the button with the English word "LOAD" can be translated as "LADDA" in Swedish.

PS: Check my 'Issue' for the correct translations - [GRAPHICAL MENU] Translation of the in-game menus #9

**Comment by Fabrice-TIERCELIN** [comment]
No problem, I'm preparing the images this way. Currently, I'm waiting for text-free images that would help the translations.

**Comment by Fabrice-TIERCELIN** [comment]
@Laserlicht, can I have the same images as [this](https://github.com/vcmi-mods/h3-themes/blob/vcmi-1.5/h3-themes/mods/AB/content/data/LOADGAME.png) but with the text "NYTT SPEL" and with the text "LADDA SPEL"?

**Comment by Laserlicht** [comment]
@Fabrice-TIERCELIN Sadly I don't know who created it. But in general: Shouldn't be that hard to create in similar style.

**Pull request merged**